int-tests/002-update.t

```diff
@@ -1,45 +0,0 @@
-Create a group to play around with.
```

Comment:
Because it was super borked on my branch, and it's still present in your branch. My plan was to merge this, and then merge yours.